test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -125,7 +111,8 @@ public void addCommand_emptyAddressBook_addressBookContainsPerson() {
 
         assertTrue(people.contains(p));
         assertEquals(1, people.immutableListView().size());
-        assertFalse(result.getRelevantPersons().isPresent());
+        //assertFalse(result.getRelevantPersons().isPresent());// result now have a component of relevant person
```

Comment:
Good job for updating JUnit tests!

On a side note, instead of commenting out unused code, delete it. We have RCS in case we need to access it again.